cmd/uhc/cluster/list/list.go

```diff
@@ -124,6 +120,12 @@ func run(cmd *cobra.Command, argv []string) {
 		managed = false
 	}
 
+	// if there is a parameter specified, assume its a filter
```

Comment:
I try to stick to comments that start with upper case and end with punctuation: `If there ... a filter:`. Can you do the same?

Reply:
Comment fixed